tests/foreman/ui/test_gpgkey.py

```diff
@@ -393,10 +368,172 @@ def test_negative_update_2(self, new_name):
                             (common_locators["alert.error"]))
             self.assertIsNone(self.gpgkey.search(new_name))
 
-    # Product association
+    # Content
```

Comment:
I think much of the boilerplate code can be dropped from these test stubs. As an example, see module [`tests.foreman.api.test_hostgroup`](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_hostgroup.py).